src/codecov-clojure/src/orb.yml

```diff
@@ -0,0 +1,68 @@
+version: 2.1
```

Comment:
In a destructured orb there should be only an @orb.yml file containing only the description and any imported orbs. All commands, executors, examples, and jobs should be in separate files in their respected folders.

Example: https://github.com/CircleCI-Public/orb-starter-kit/tree/master/src